rename_files.sh

```diff
@@ -94,7 +94,7 @@ while read -r a_file_name; do
   # a variable.
   # -e - flag for indicating the pattern you want to match against.
 
-  if (echo "$a_file_name" | grep -q -e "$old_text"); then
+  if (echo basename "$a_file_name" | grep -q -e "$old_text"); then
     echo "FOUND $old_text IN:" "$(basename "$a_file_name")"
     new_file_name="${a_file_name/$old_text/$new_text}"
     mv "$a_file_name" "$new_file_name"
```

Comment:
I think there is a potential bug here. If `$old_text` appears in _both_ the directory part and the base name part of `$a_file_name`, you'll change the occurrence in the directory name part. Suggest capturing in variables the directory name and base name parts of `$a_file_name` and working with those.

Reply:
@kcharter Added some variables, re-built correct path & filename for renaming & re-tested.

Reply:
Everything looks good to me @kimlew